src/raygun.js

```diff
@@ -451,8 +451,11 @@ var raygunFactory = function(window, $, undefined) {
       error = event.reason.error;
     }
     if (!error) {
-      error = event;
+      error = new Error('Unhandled promise rejection');
+      // Clear the stacktrace, as we don't want the error to appear to come from raygun4js
+      error.stack = null;
```

Comment:
I suppose there is no way to know which file/line/column number the error came from and use that in the stack trace? As a developer it would really help when debugging these sorts of errors is all :)

Reply:
I've had a good dig through the `event` object (and the promise itself) looking for anything which would provide a clue as to where the error occurred. I haven't been able to find anything - but happy to have another pass at this, as I agree that it would be very helpful.